docker/Dockerfile

```diff
@@ -43,9 +43,9 @@ RUN echo "Enable server extensions" \
   && jupyter serverextension enable --py --sys-prefix jupyterlab_iframe \
   && echo done
 
-COPY requirements-odc.txt /conf/
-RUN echo "Adding odc-dependencies" \
-  && env-build-tool extend /conf/requirements-odc.txt ${py_env_path}
+COPY requirements-odc-fixed.txt /conf/
```

Comment:
Why `-fixed`?

Reply:
Or does it mean 'fixed' as in 'not changing'? Maybe 'static' might be better... I read it as in fixed=repaired.

Reply:
good point, will change to `-static`?